weatherAplication/Modules/Home/ViewModel/HomeViewModel.swift

```diff
@@ -2,13 +2,29 @@ import Foundation
 
 protocol HomeViewModelContract {
     var delegate: HomeViewModelDelegate? { get set }
+    var lastSearches: [CityInfo] { get set }
 
     func proceedButtonPressed()
 }
 
 class HomeViewModel: HomeViewModelContract {
 
     var delegate: HomeViewModelDelegate?
+    var lastSearches: [CityInfo]
+
+    init() {
+        if let data = UserDefaults.standard.data(forKey: "test1") {
```

Comment:
Zmień nazwę tego klucza na bardziej opisującą to co faktycznie znaczy i wrzuć ją do stałej